.circleci/scripts/vs_install.ps1

```diff
@@ -1,15 +1,7 @@
-$VS_DOWNLOAD_LINK = "https://aka.ms/vs/15/release/vs_buildtools.exe"
+$VS_DOWNLOAD_LINK = "https://aka.ms/vs/16/release/vs_community.exe"
 $COLLECT_DOWNLOAD_LINK = "https://aka.ms/vscollect.exe"
-$VS_INSTALL_ARGS = @("--nocache","--quiet","--wait", "--add Microsoft.VisualStudio.Workload.VCTools",
-                                                     "--add Microsoft.VisualStudio.Component.VC.Tools.14.13",
-                                                     "--add Microsoft.Component.MSBuild",
-                                                     "--add Microsoft.VisualStudio.Component.Roslyn.Compiler",
-                                                     "--add Microsoft.VisualStudio.Component.TextTemplating",
-                                                     "--add Microsoft.VisualStudio.Component.VC.CoreIde",
-                                                     "--add Microsoft.VisualStudio.Component.VC.Redist.14.Latest",
-                                                     "--add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core",
-                                                     "--add Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
-                                                     "--add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Win81")
+$VS_INSTALL_ARGS = @("modify","--nocache","--quiet","--wait", "--installPath ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Community""",
```

Comment:
Avoid installing an existing toolchain would be in the to-do list here.

Reply:
Where do you get the link of prior version of VS Community?

Reply:
You may get that from the chocolatey page, e.g https://chocolatey.org/packages/visualstudio2019community/16.5.0.0. But I wonder whether it may work since we already have a updated version of installer in the image.

Reply:
I think this is a disaster, the MSVC toolchain is not so stable, but they love to get thing updated automatically. What's more, they didn't provide a way to get the legacy builds (at least for Community).

Reply:
So it looks like adding a component on an existing VS2019 installation won't work if you want to pin the VC version to a revision level.